internal/db/db.go

```diff
@@ -61,9 +60,31 @@ func (db *Db) Close() {
 	db.sqlDb.Close()
 }
 
+func splitStatementToPieces(statementsString string) (pieces []string, err error) {
+  pieces = make([]string, 0, 16)
+  embedded := map[rune]bool{'\'': false, '"': false}
+	stmtBegin := 0
+  var stmt string
+  for i, char := range statementsString {
+    if status, present := embedded[char]; present {
+      embedded[char] = !status
+    }
+    if embedded['\''] || embedded['"'] || char != ';' {
+      continue
+    }
+    stmt = statementsString[stmtBegin : i]
```

Comment:
since we're keeping the semicolumn (`;`) for the last statement, maybe we should keep for other ones as well. What do you think?
```suggestion
    stmt = statementsString[stmtBegin : i+1]
```